tests/replicate_users_meta.js

```diff
@@ -0,0 +1,40 @@
+const expect = require('chai').expect,
+      PouchDB = require('../libs/db'),
+      singleMedicDoc = require('./docs/single-medic.json'),
+      pgutils = require('./utils/pgutils'),
+      replicate = require('../libs/replicate');
+
+const couchUrl = `${process.env.TEST_COUCH_URL}/mycouch`;
+const pgDbName = 'pgsinglerecordtest';
+const pgUrl = `${process.env.TEST_PG_URL}/${pgDbName}`;
+
+const pouch = () => new PouchDB(`${couchUrl}-users-meta`);
+
+const cleanUp = async () => {
+  await pouch().destroy();
+  await pgutils.ensureDbIsClean(pgUrl);
+};
+
+describe('medic users meta db replication', () => {
+  let pg;
+
+  beforeEach(async () => {
+    await cleanUp();
+    await pouch().bulkDocs([singleMedicDoc]);
+    pg = new pgutils.Pg(pgUrl);
+  });
+
+  afterEach(async() => {
+    await pg.destroy();
+  });
+
+  it('replicates single couch record to the right table on postgres', async() => {
+    await replicate(couchUrl, pgUrl, {timesToRun: 1, couchdbUsersMeta: true});
+    let rows = await pg.rows('couchdb_users_meta');
+    expect(rows.length).to.equal(1);
+    const [couchRecord, pgRecord] = [singleMedicDoc, rows[0].doc];
+    expect(pgRecord.sex).to.equal(couchRecord.sex);
+    expect(pgRecord.role).to.equal(couchRecord.role);
+    expect(pgRecord.type).to.equal(couchRecord.type);
+  });
+});
```

Comment:
Ideally we'd also be testing the migration and materialised view. Do you think it would be easy enough to execute that and query the view to ensure it's outputting the rows we expect?

Reply:
Yes. I have added a test for the material views an the migration.